features/xpath.yml

```diff
@@ -0,0 +1,25 @@
+name: XPath
+description: "XML Path Language (`XPath`) uses a syntax akin to file paths to provide a flexible way of pointing to or selecting different parts of an XML document. It can also be used to test addressed nodes within a document to determine whether they match a pattern or not."
+spec: https://dom.spec.whatwg.org/#xpath
+group: xml
+compat_features:
+  - api.Document.evaluate
+  # Doesn't really fit well here, maybe should stand alone,
+  # or be part of DOM?
+  #- api.XMLDocument
```

Comment:
I agree, this should be part of the core DOM feature(s). I _think_ you get an `XMLDocument` if you have an XHTML doctype, so it's still "just" a document for the purposes of DOM.